scripts/README.md

```
@@ -0,0 +1,15 @@
+## Overview
+This folder is a collection of the scripts running various services on the Pis.
+The extensions ending in .service are systemd files. Systemd is a way of taking programs (bash/python in this case)
+and converting them into autostarting services.
+
+## To install:
+#### Move files ending in .sh to the regular home directory (~ which is /home/robo)
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)